hasher-matcher-actioner/hmalib/lambdas/api/submit.py

```diff
@@ -153,6 +160,28 @@ def record_content_submission(dynamodb_table: Table, request: SubmitContentReque
     ).write_to_table(dynamodb_table)
 
 
+def submit_from_url(
```

Comment:
This is the method that is unnecessarily HTTP friendly. Make this pythonic. can raise exceptions and use primitive arguments. 

Restrict the http request / response payload / status code things to the `from_url` method below.

Reply:
The benefit is that the function invocation from python becomes more natural. Right now, we have

`submit_from_url()` catches exception, returns SubmitContentError -> api_root's `s3_event_handler` checks for instance of SubmitContentError and then raises exception.

This is happening because submit_from_url() is trying to be HTTP friendly. It need not.

Reply:
see above